src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -53,79 +54,79 @@
 
 	private static byte[] patchworkGreyscaleIcon;
 
-	static {
-		try {
-			patchworkGreyscaleIcon = Files.readAllBytes(Paths.get(Patchwork.class.getResource("/patchwork-icon-greyscale.png").toURI()));
-		} catch (IOException | URISyntaxException e) {
-			e.printStackTrace();
-		}
+	private Path inputDir, outputDir, dataDir, tempDir;
+	private IMappingProvider primaryMappings;
+	private List<IMappingProvider> devMappings;
+	private NaiveRemapper naiveRemapper;
+	private ManifestRemapper manifestRemapper;
+	private boolean closed = false;
 
+	static {
 		// TODO: With the new logger from application-core, this is
 		// 		 a little problem, since it does not follow the concept of
 		//		 component sub loggers (see Logger#sub)
 		LOGGER = new Logger("Patchwork");
 		LOGGER.setWriter(new StreamWriter(true, System.out, System.err), LogLevel.TRACE);
-	}
-
-	public static void main(String[] args) throws Exception {
-		File current = new File(System.getProperty("user.dir"));
-		Path currentPath = current.toPath();
-		File voldemapTiny = new File(current, "data/mappings/voldemap-" + version + ".tiny");
-		List<TsrgClass<RawMapping>> classes = Tsrg.readMappings(new FileInputStream(new File(current, "data/mappings/voldemap-" + version + ".tsrg")));
 
-		IMappingProvider intermediary = TinyUtils.createTinyMappingProvider(currentPath.resolve("data/mappings/intermediary-" + version + ".tiny"), "official", "intermediary");
-		TsrgMappings mappings = new TsrgMappings(classes, intermediary);
-
-		if (!voldemapTiny.exists()) {
-			TinyWriter tinyWriter = new TinyWriter("official", "srg");
-			mappings.load(tinyWriter);
-			String tiny = tinyWriter.toString();
-			Files.write(voldemapTiny.toPath(), tiny.getBytes(StandardCharsets.UTF_8));
+		try {
+			patchworkGreyscaleIcon = Files.readAllBytes(new File(Patchwork.class.getResource("/patchwork-icon-greyscale.png").getPath()).toPath());
```

Comment:
Couldn't this just use Paths.get directly?

Reply:
Wouldn't Paths.get try to get the path in the working directory, not in the jar file?

Reply:
But new File() does the same thing, no?

Reply:
Wait we're talking about something else, never mind.